src/main/java/com/example/demo/HelloWorldController.java

```diff
@@ -0,0 +1,19 @@
+package com.example.demo;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloWorldController {
+
+    @RequestMapping
```

Comment:
Let's use `@GetMapping` or `@PostMapping`

Reply:
Sure, I made the change